css/index.css

```diff
@@ -0,0 +1,79 @@
+body{
+  background-color: lavender;
+  color:indigo;
+  font-style: quicksand;
```

Comment:
It looks like you were going for the extra step of bringing in a Google font here, but what shows on the page is still the default browser font.  Getting the font imported properly is just a couple more steps. On the [Google page](https://fonts.google.com/specimen/Quicksand?query=quicksand) for the font you want, just click the "Select this style" next to the version/font-weight you want:
 ![Screenshot from 2021-11-17 20-23-11](https://user-images.githubusercontent.com/77859694/142335624-790b63ad-e033-432e-87d6-4ba4f14b1f46.png)

That will make this menu pop out from the side:
![Screenshot from 2021-11-17 20-23-44](https://user-images.githubusercontent.com/77859694/142335626-50b2bd51-30e5-44fe-a204-14ee8e387991.png)

You'll need to copy that series of links under the `<link>` radio button, and paste it into the `<head>` area of your HTML (just like how you linked your CSS stylesheet). Then you'll need to copy and paste the CSS that starts with `font-family` into your CSS, in place of the `font-style` property you have here. [font-style](https://www.w3schools.com/cssref/pr_font_font-style.asp) generally isn't used for much beyond italicizing stuff.